demo/src/app/components/+tabs/demos/custom-template/custom-template.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <tabset>
+    <tab heading="Tab 1">Tab content</tab>
+    <tab (select)="alertMe()">
```

Comment:
perhaps there should be a bit more tabs with custom templates. for example, first is triggering alert, second has some html tags in it, third has an icon

Reply:
changed